flask_praetorian/base.py

```diff
@@ -722,24 +723,27 @@ def read_token_from_cookie(self):
 
     def read_token(self):
         """
-        Tries to unpack the token from the current flask request in the locations configured with JWT_PLACES
+        Tries to unpack the token from the current flask request
+        in the locations configured by JWT_PLACES.
         Check-Order is defined by the value order in JWT_PLACES.
         """
         for place in self.jwt_places:
             try:
-                return getattr(self, "read_token_from_{place}".format(place=place.lower()))()
+                return getattr(self, "read_token_from_{place}"
+                                     .format(place=place.lower()))()
             except MissingToken:
                 pass
             except AttributeError:
-                flask.current_app.logger.warning("Flask_Praetorian hasn't implemented reading JWT tokens from location {place}. Please reconfigure JWT_PLACES.".format(place=place.lower()))
-                flask.current_app.logger.warning("Values accepted in JWT_PLACES are: {def_places}".format(def_places=DEFAULT_JWT_PLACES))
-
-        raise MissingToken("Could not find token in any of the given locations: {locations}".format(
-            locations=self.jwt_places,
-        ))  
-        
-        
-        
+                current_app.logger.warning("Flask_Praetorian hasn't implemented reading JWT tokens from location {place}. Please reconfigure JWT_PLACES."
```

Comment:
instead of importing current_app separately, please just use `flask.current_app`

Reply:
Will do